doghouse/server/doghouse.go

```diff
@@ -0,0 +1,211 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/bradleyfalzon/ghinstallation"
+	"github.com/google/go-github/github"
+	"github.com/haya14busa/reviewdog"
+	"github.com/haya14busa/reviewdog/diff"
+	"github.com/haya14busa/reviewdog/doghouse"
+)
+
+type DogHouse struct {
+	client *http.Client
+	req    *doghouse.CheckRequest
+	gh     *github.Client
+}
+
+func New(req *doghouse.CheckRequest, privateKey []byte, integrationID int, c *http.Client) (*DogHouse, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function New should have comment or be unexported